ports/wolfmqtt/portfile.cmake

```
@@ -1,9 +1,11 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO wolfssl/wolfmqtt
-    REF v1.13.0
-    SHA512 2bd05471c548bacf6fc5166d9c2ee546b3ce5c4fab89ff16175de9ab7696b8b570cdd7d4377378fc0cb579447488dbf727cb938192eb3180ed007414cb949a8a
+    REF v1.14.0
+    SHA512 9449d87c543e823b3517a4605343a92207499812e75c950a2e8fb3d969333d39579dac69657e298826ba65017a8208c28934d7c6a4abbb4bf308514047273191
     HEAD_REF master
+    PATCHES
+      wolfmqtt_pr305.diff
```

```suggestion
      wolfmqtt_pr305.patch
```
Sorry for the late reply. Please change the patch to the unified naming format `xx.patch`. :)

---

This does not need to be changed, `.diff` is correct.